.github/workflows/ui-tests.yml

```diff
@@ -94,14 +100,14 @@ jobs:
         uses: actions/upload-artifact@v4
         if: ${{ failure() }}
         with:
-          name: ui-tests-ios-swiftui-${{matrix.device}}.xcresult
```

Comment:
`m`: We should keep some value of the matrix in this name, otherwise when adding another matrix entry the upload will warn/fail with a "duplicate artifact name" error

Reply:
I removed the matrix.